src/main/java/Duke/Main/Ui.java

```diff
@@ -0,0 +1,63 @@
+package Duke.Main;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    private final Scanner sc;
+
+    /**
+     * Constructor for the UI class, setting up a Scanner object
```

Comment:
According to the standard in this course, it is better to add a full stop at the end of the brief description.